app/workers/start_of_cycle_notification_worker.rb

```diff
@@ -0,0 +1,59 @@
+class StartOfCycleNotificationWorker
+  include Sidekiq::Worker
+
+  def perform(service, batch_size = 100)
+    @service = service
+
+    providers_scope.find_each(batch_size: batch_size) do |provider|
+      provider.provider_users.where.not(Arel.sql(chaser_sent_sql)).each do |provider_user|
+        ProviderMailer.send(mailer_method, provider_user)
+        ChaserSent.create!(chased: provider_user, chaser_type: mailer_method)
+
+        next if service == :apply
+        next unless provider_user.provider_permissions.find_by(provider: provider).manage_organisations
```

Comment:
There are some methods that could be helpful in `ProviderAuthorisation` and `ProviderSetup`, which we use to calculate `relationship_pending?`, wonder if we could reuse those instead here?